index.html

```diff
@@ -19,8 +19,176 @@ <h1>Sebastián<br>Muñoz</h1>
     <section class="intro">
         <p>Hello! I am a software developer! I can help you build a product, feature or website.      Taka a look of my works. <br> If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
     </section>
-    <div class="collab">
+    <div class="btn">
         <button type="button">Start collaboration</button>
     </div>
+    <div class="geometry">
+        <img src="assets/img/image_geometry_2.png" alt="">
+    </div>
+    <section class="projects-title">
+        <h1>Projects</h1>
+        <ul class="projects">
+            <li>
+                <div class="grid-container">
+                    <div class="picture">
+                        <img src="assets/img/Project1.png" alt="">
+                    </div>
+                    <div class="project-grid-item">
+                        <a href="" class="project-name">Project name goes here</a> 
+                    </div>
+                    <ul class="project-lang">
+                        <li>HTML/CSS</li>  
+                        <li>Ruby on Rails</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <div class="project-btn-grid btn">
+                        <button type="button" class="projects-btn">See this project <img src="assets/img/Union.png" alt=""></button>
+                    </div>
+                </div>
+            </li>
+            <li>
+                <div class="grid-container">
+                    <div class="picture">
+                        <img src="assets/img/Project2.png" alt="">
+                    </div>
+                    <div class="project-grid-item">
+                        <a href="" class="project-name">Project name goes here</a> 
+                    </div>
+                    <ul class="project-lang">
+                        <li>HTML/CSS</li>  
+                        <li>Ruby on Rails</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <div class="project-btn-grid btn">
+                        <button type="button" class="projects-btn">See this project <img src="assets/img/Union.png" alt=""></button>
+                    </div>
+                </div>
+            </li>
+            <li>
+                <div class="grid-container">
+                    <div class="picture">
+                        <img src="assets/img/Project3.png" alt="">
+                    </div>
+                    <div class="project-grid-item">
+                        <a href="" class="project-name">Project name goes here</a> 
+                    </div>
```

Comment:
- [ ] For Semantic's sake this `div` tag can be gracefully replaced by `hx` tag.